scalableminds/webknossos
# Make version history view paginated

### URL of deployed dev instance (used for testing):
- [paginationforupdateactionlog.webknossos.xyz](https://padingationforupdateactionlog.webknossos.xyz/)

### Steps to test:
- Create some annotations
- Check the update history

### Issues:
- fixes #5421 

------
